lib/ts/BufferWriter.h

```diff
@@ -652,6 +652,22 @@ bwformat(BufferWriter &w, BWFSpec const &, char c)
   return w.write(c);
 }
 
+inline BufferWriter &
+bwformat(BufferWriter &w, BWFSpec const &spec, bool f)
+{
+  static const ts::string_view txt_true{"true"};
+  static const ts::string_view txt_false{"false"};
+  static const ts::string_view txt_TRUE{"TRUE"};
+  static const ts::string_view txt_FALSE{"FALSE"};
+  if ('s' == spec._type)
+    w.write(f ? txt_true : txt_false);
```

Comment:
Why not just:

f ? "true" : "false"

And were are all the obligatory superfluous braces?